pkg/provider/gcp/secretmanager/secretsmanager.go

```diff
@@ -64,6 +65,8 @@ type GoogleSecretManagerClient interface {
 	Close() error
 }
 
+var useMu = sync.Mutex{}
```

Comment:
Please add a comment to decrease the number of WTFs when reading the code again in 12 Months from now :joy: